index.html

```diff
@@ -28,7 +28,7 @@ <h1>Dr.Norman Borlaug</h1>
       <figure id="img-div" class="">
         <img src="./image-asset.jpeg" alt="" id="image" class="img-fluid" />
         <figcaption id="img-caption">
-          <span class="fw-bold"
+          <span class="fw-bold" id="fw-bold"
```

Comment:
Kindly remove the id attribute in the span class. unused codes make files look messy.